index.html

```diff
@@ -0,0 +1,340 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta
+        name="viewport"
+        content="width=device-width, initial-scale=1.0"
+    >
+    <title>Projekt-01</title>
+
+
+    <link
+        rel="preconnect"
+        href="https://fonts.googleapis.com"
+    >
+    <link
+        rel="preconnect"
+        href="https://fonts.gstatic.com"
+        crossorigin
+    >
+    <link
+        href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap"
+        rel="stylesheet"
+    >
+    <!-- <link
+        rel="stylesheet"
+        href="./css/reset.css"
+    > -->
+    <link
+        rel="stylesheet"
+        href="./css/main.css"
+    >
+
+</head>
+
+<body>
+
+    <header class="header main-container">
+        <div
+            class="navbar__container container"
+            id="nav"
+        >
+            <img
+                class="header__logo"
+                src="./images/logo.svg"
+                alt="header-logo-nova"
+            >
+            <nav class="nav">
+                <ul class="nav__items">
+                    <li class="nav__links"><a
+                            class="nav__link"
+                            href="#team"
+                        >The Team</a></li>
+                    <li class="nav__links"><a
+                            class="nav__link"
+                            href="#offer"
+                        >Pricing</a></li>
+                    <li class="nav__links"><a
+                            class="nav__link"
+                            href="#features"
+                        >Features</a></li>
+                </ul>
+            </nav>
+        </div>
+
+        <section class="hero container">
+            <div class="hero__info">
+                <h1 class="hero__header">Beautiful Free Nova template</h1>
+                <p class="hero__sub-header">A top notch premium quality <br> template for your next <br> project.</p>
+                <button class="cta-button">Download FREE</button>
+            </div>
+            <div class="hero__image">
+                <div class="hero__image-container">
+
+                </div>
+            </div>
+        </section>
+    </header>
+    <main>
+
+        <section
+            class="features main-container"
+            id="features"
+        >
+            <h2 class="features__heading container">Features</h2>
+            <div class="features__content container">
+                <div class="feature">
```

Comment:
Osobiście wrzuciłbym tutaj `ul > li`, gdzie `li` to "feature" bo mamy tutaj "listę" cech więc idealnie to pasuje ;P